addons/html_editor/static/src/main/font/font_plugin.js

```diff
@@ -443,28 +446,35 @@ export class FontPlugin extends Plugin {
     }
 
     /**
-     * Transform an empty heading, blockquote or pre at the beginning of the
-     * editable into a paragraph.
+     * Transform an empty heading, blockquote, pre or signature container
+     * at the beginning of the editable into a paragraph.
      */
     handleDeleteBackward({ startContainer, startOffset, endContainer, endOffset }) {
         // Detect if cursor is at the start of the editable (collapsed range).
         const rangeIsCollapsed = startContainer === endContainer && startOffset === endOffset;
         if (!rangeIsCollapsed) {
             return;
         }
-        // Check if cursor is inside an empty heading, blockquote or pre.
-        const closestHandledElement = closestElement(endContainer, handledElemSelector);
-        if (!closestHandledElement || closestHandledElement.textContent.length) {
+        const block = closestBlock(endContainer);
+        // Check if cursor is inside an empty block.
+        if (!block || !isEmptyBlock(block)) {
```

Comment:
`isEmptyBlock` checks for `blockEl`.
```suggestion
        if (!isEmptyBlock(block)) {
```